tests/app/views/contexts/test_list_collector_context.py

```diff
@@ -1,45 +0,0 @@
-from app.views.contexts.list_collector import _generate_list_item_title
```

Comment:
I realise the views are difficult to test, but there doesn't appear to be existing unit tests that cover the list collector context class?